### PR TITLE
Refactor structure

### PR DESCRIPTION
**Pull request by acodeninja**
- change name of module folder as `commands` may conflict with other site packages
- move commands into separate module and change to consistent names

### CODE REVIEW

**Review by yusufsheiqh — COMMENTED** [review]
I see we have used `dbt_copilot_helper`, but I think it would make more sense to use `copilot_tools` or `dbt_copilot_tools` instead as that is the package name. The script entrypoint is kept as `copilot-helper` for convenience as we use `copilot` CLI as well.

File: tests/test_command_bootstrap.py [hunk 1]
```diff
@@ -107,6 +107,8 @@ def test_make_config(tmp_path):
     switch_to_tmp_dir_and_copy_config_file(tmp_path, "test_config.yml")
     os.mkdir(f"{tmp_path}/copilot")
 
+    print(os.listdir(tmp_path))
```
Comment (WillGibson) [inline tests/test_command_bootstrap.py:110]: Did you mean to leave this in?

**Comment by acodeninja** [comment]
> I see we have used `dbt_copilot_helper`, but I think it would make more sense to use `copilot_tools` or `dbt_copilot_tools` instead as that is the package name. The script entrypoint is kept as `copilot-helper` for convenience as we use `copilot` CLI as well.

I did think about this, can move to `dbt_copilot_tools` but my thought was that the command is `copilot_helper` so it made sense to keep that name going, as conceptually `copilot_helper` and associated commands is part of the dbt-copilot-tools package.

**Comment by yusufsheiqh** [comment]
> > I see we have used `dbt_copilot_helper`, but I think it would make more sense to use `copilot_tools` or `dbt_copilot_tools` instead as that is the package name. The script entrypoint is kept as `copilot-helper` for convenience as we use `copilot` CLI as well.
> 
> I did think about this, can move to `dbt_copilot_tools` but my thought was that the command is `copilot_helper` so it made sense to keep that name going, as conceptually `copilot_helper` and associated commands is part of the dbt-copilot-tools package.

@acodeninja I think I prefer `dbt_copilot_tools` in that case, although the `dbt` prefix may be redundant. I really like the dropping of the `_cli` suffix for the commands.

Another point to bring up... we are using `utils.py` and the `utils` folder, but I think it would be worthwhile if we could consolidate both of these as part of this restructure as they are both used for `dbt_copilot_tools`.

**Comment by WillGibson** [comment]
Is now the time to clarify the purpose of this repository and possibly rename it?

Is it expecting to include more tools than `copilot-helper`?

If it is, would we publish them all under the "tools` package or perhaps individually?

Currently as it is only `copilot-helper` and no sign of it being something else too, I keep having a desire to rename the repository and package to `dbt-copilot-helper`.

**Review by WillGibson — APPROVED** [review]

**Comment by yusufsheiqh** [comment]
The repository name makes sense to me, since we do have migration scripts and files such as the `bootstrap-config` files.

But, it might be time to rename the package to `dbt-copilot-helper` as we only package up the things required for our `copilot-helper` script entrypoint such as templates, commands, etc.

**Review by yusufsheiqh — APPROVED** [review]
Let's get this merged as this PR makes sense and move forwards based on the discussions we are having. 🚀

**Pull request merged**